src/main/java/io/arivera/oss/embedded/rabbitmq/EmbeddedRabbitMqConfig.java

```diff
@@ -446,9 +456,14 @@ public EmbeddedRabbitMqConfig build() {
           erlangCheckTimeoutInMillis,
           cacheDownload, deleteCachedFile,
           envVars,
-          processExecutorFactory);
+          processExecutorFactory,
+          downloadProxy);
     }
 
+    public Builder downloadProxy( String hostname, int port ) {
```

Comment:
i made it like that becouse embedded mongo has the helper method, but yeah seems better to also have a method that takes a Proxy argument, i would like to keep both